src/components/app.tsx

```diff
@@ -79,8 +82,8 @@ export class App extends React.PureComponent<IProps, IState> {
       this.setState(newState as IState);
 
       if (teacherEditionMode) {
-        createPluginNamespace();
-        loadPluginScripts(activity);
+        this.LARA = initializeLara();
```

Comment:
I don't particularly see this as a pain point at the moment. It's annoying to have to rely on a `LARA` global variable, but I've set it up so that the AP itself doesn't rely on the global but the plugins still can.